<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
    ~~~~~~~~~~~~~~~~^
        recipe,
        ^^^^^^^
        fix=fix,
        ^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
    ~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/check_syntax.py", line 36, in check_recipe
    has_constraints = constraints.search(spec)
TypeError: expected string or bytes-like object, got 'NoneType'

ERROR conda.cli.main_run:execute(127): `conda run conda-lint /tmp/abs_71v56vo2no/clone` failed. (See above for error)
</details>

--- a/recipe/bld.bat
+++ b/recipe/bld.bat
@@ -8,11 +8,10 @@ if "%ARCH%" == "32" (
 )
 
 :: Set temp folder for the outputs
-set TMP_FOLDER=%LIBRARY_PREFIX%\TEMP
+set ARTIFACTS_DIR=%LIBRARY_PREFIX%\TEMP
 cd /d %SRC_DIR%\builds\msvc\vs%VS_YEAR%\
-msbuild libsodium.sln /p:Configuration=DynRelease /p:Platform=%ARCH% /p:OutDir=%TMP_FOLDER%\dynamic\
+msbuild libsodium.sln /p:Configuration=DynRelease /p:Platform=%ARCH% /p:OutDir=%ARTIFACTS_DIR%\
 if errorlevel 1 exit 1
-set ARTIFACTS_DIR=%TMP_FOLDER%
 
 :: Check for built DLLs
 if not exist %ARTIFACTS_DIR%\libsodium.dll    exit 1

--- a/recipe/bld.bat
+++ b/recipe/bld.bat
@@ -1,50 +1,24 @@
 @echo on
 
+:: Pick the correct architecture
 if "%ARCH%" == "32" (
   set ARCH=Win32
 ) else (
   set ARCH=x64
 )
 
-:: See https://github.com/conda-forge/staged-recipes/pull/194#issuecomment-203577297
-:: Nasty workaround. Need to move a more current msbuild into PATH.  The one on
-:: AppVeyor barfs on the solution. This one comes from the Win7 SDK (.net 4.0),
-:: and is known to work.
-if %VS_MAJOR% == 9 (
-    COPY C:\Windows\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe .\
-    set "PATH=%CD%;%PATH%"
-)
+:: Set temp folder for the outputs
+set TMP_FOLDER=%LIBRARY_PREFIX%\TEMP
+cd /d %SRC_DIR%\builds\msvc\vs%VS_YEAR%\
+msbuild libsodium.sln /p:Configuration=DynRelease /p:Platform=%ARCH% /p:OutDir=%TMP_FOLDER%\dynamic\
+if errorlevel 1 exit 1
+set ARTIFACTS_DIR=%TMP_FOLDER%
 
-if "%VS_YEAR%" == "2008" (
-  xcopy /s /y /i %RECIPE_DIR%\vs2008 %SRC_DIR%\builds\msvc\vs%VS_YEAR%
-  copy %LIBRARY_INC%\stdint.h %SRC_DIR%\builds\msvc\
-  copy %LIBRARY_INC%\inttypes.h %SRC_DIR%\builds\msvc\
-  cd /d %SRC_DIR%\builds\msvc\vs%VS_YEAR%\dynamic
-  msbuild libsodium.sln /p:Configuration=Release /p:Platform=%ARCH%
-  if errorlevel 1 exit 1
-  cd /d %SRC_DIR%\builds\msvc\vs%VS_YEAR%\static
-  msbuild libsodium.sln /p:Configuration=Release /p:Platform=%ARCH%
-  if errorlevel 1 exit 1
-  :: Generate version.h
-  cd /d %SRC_DIR%
-  call msvc-scripts\process.bat
-  if errorlevel 1 exit 1
-  set ARTIFACTS_DIR=%SRC_DIR%\bin\%ARCH%\Release\v90
-) else (
-  :: Temp folder for the outputs
-  set TMP_FOLDER=%LIBRARY_PREFIX%\TEMP
-  cd /d %SRC_DIR%\builds\msvc\vs%VS_YEAR%\
-  msbuild libsodium.sln /p:Configuration=DynRelease /p:Platform=%ARCH% /p:OutDir=%TMP_FOLDER%\dynamic\
-  if errorlevel 1 exit 1
-  msbuild libsodium.sln /p:Configuration=StaticRelease /p:Platform=%ARCH% /p:OutDir=%TMP_FOLDER%\static\
-  if errorlevel 1 exit 1
-  set ARTIFACTS_DIR=%TMP_FOLDER%
-)
+:: Check for built DLLs
+if not exist %ARTIFACTS_DIR%\libsodium.dll    exit 1
 
-if not exist %ARTIFACTS_DIR%\dynamic\libsodium.dll    exit 1
-
-move /y %ARTIFACTS_DIR%\dynamic\libsodium.dll %LIBRARY_BIN%
-move /y  %ARTIFACTS_DIR%\dynamic\libsodium.lib %LIBRARY_LIB%
-move /y %ARTIFACTS_DIR%\static\libsodium.lib %LIBRARY_LIB%\libsodium_static.lib
+:: Copy dynamic lib, import lib and headers to their correct destinations
+move /y %ARTIFACTS_DIR%\libsodium.dll %LIBRARY_BIN%
+move /y  %ARTIFACTS_DIR%\libsodium.lib %LIBRARY_LIB%
 xcopy /s /y /i %SRC_DIR%\src\libsodium\include\sodium %LIBRARY_INC%\sodium
 xcopy /s /y %SRC_DIR%\src\libsodium\include\sodium.h %LIBRARY_INC%\

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -37,7 +37,6 @@ test:
     - if not exist %LIBRARY_INC%\sodium\version.h      exit 1  # [win]
     - if not exist %LIBRARY_BIN%\libsodium.dll         exit 1  # [win]
     - if not exist %LIBRARY_LIB%\libsodium.lib         exit 1  # [win]
-    - if not exist %LIBRARY_LIB%\libsodium_static.lib  exit 1  # [win]
 
 about:
   home: https://libsodium.org

